src/Pagination.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import BootstrapMixin from './BootstrapMixin';
+
+const Pagination = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    activePage: React.PropTypes.number,
+    items: React.PropTypes.number,
```

Comment:
Is it possible for this to be an array of links?

Or possibly include a prop for `hrefTemplate` were the page number can be inserted. Thus allowing this to work with non-SPA sites.